### PR TITLE
Add basic customization page

### PR DESCRIPTION
**Pull request by splondike**
Adds a new page targeted at customisation for beginner/non programmer audiences.

### CODE REVIEW

**Comment by splondike** [comment]
If you'd like to read the formatted page you can look at this PDF: [basic-customization.pdf](https://github.com/TalonCommunity/Wiki/files/7103525/basic-customization.pdf)

**Review by didoesdigital — APPROVED** [review]
The new basic customization page is fantastic! Thanks for writing all of this. I really dig the approach of concentrating on what you can achieve with just Talon files without getting tangled up in Python files, so it's easier to get the hang of making Talon work for you.

I added some formatting suggestions and asked some questions but this is already a great resource to get into the hands of beginners ASAP!

**Comment by isabella-grigg-eyley** [comment]
This looks really great! I think the only feedback I have is that it would be handy to have quick jump links at the top to the actions and keypress syntax lists. Otherwise looks good!

File: basic_customization.md [hunk 1]
```diff
@@ -0,0 +1,238 @@
+---
+sidebar: true
+order: 2
+published: true
+---
+# Basic customization
+
+Once you have successfully [set up Talon](/getting_started) you may find that you would like to change some of how it behaves. This page aims to be a pragmatic guide for performing some common modifications and is aimed at a beginner audience, including non-programmers. A more complete treatment of Talon's capabilities can be found in the [unofficial Talon docs](/unofficial_talon_docs). This page assumes that Talon is responding to your voice and you are using the knausj\_talon package.
+
+If you are having trouble with anything here or in Talon more generally, the best way to get help is on the [Talon Slack](/).
```
[the remark this answers is not in the document]
Comment (splondike) [reply basic_customization.md:10]: This is intentional, there's discussion of the Talon slack on the index page.

File: basic_customization.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+
+Given this flexibility there are two ways you could approach customizing your Talon setup. The first is just to edit the `.talon` and `.py` files within the knausj\_talon package/directory directly. The second is to maintain your own separate directory with your customizations. The first can be easier to begin with, but makes it somewhat harder to keep your knausj\_talon up to date. This is because you need to work out what you modified and how to re-apply it to the knausj package every time you update it. We recommend using the separate directory approach as much as possible, only resorting to direct editing when you have to.
+
+The screenshot below shows my talon user directory. You can see that I have an engines.py and a knausj\_talon folder in the same way that you would. The rest of the folders are other 'packages' I'm using. All my personal customizations are in the 'mystuff' folder; knausj\_talon is completely unchanged from what I downloaded from github.
+
+![Screen shot of Talon user directory](/media/basic_customization_folders.png)
+
```
[the remark this answers is not in the document]
Comment (splondike) [reply basic_customization.md:23]: Have re-ordered the paragraphs a bit in my version, which should address this.

**Comment by splondike** [comment]
OK, I've processed everybody's feedback on my local version. I'll have a final read over it tomorrow and will then publish it to talon.wiki.

FYI Isabella, there will be links to the first two levels of headers put in the sidebar automatically. This should include the two sections you indicated.

**Pull request merged**